securedrop/db.py

```diff
@@ -190,6 +193,18 @@ class BadTokenException(Exception):
     """Raised when a user logins in with an incorrect TOTP token"""
 
 
+class InvalidPasswordLength(Exception):
+    """Raised when attempting to create a Journalist or log in with an invalid
+    password length"""
+
+    def __init__(self, password):
+        self.password = password
+
+    def __str__(self):
+        if len(password) > Journalist.MAX_PASSWORD_LEN:
```

Comment:
If statement will throw `NameError: global name 'password' is not defined`. Should be `len(self.password)`.